### PR TITLE
Patient model changes: DCAF soft pledge => fund pledge, resolved without dcaf => resolved without fund

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

This changes `patient.resolved_without_dcaf` to `patient.resolved_without_fund`, and `patient.dcaf_soft_pledge` to `patient.fund_pledge`. And makes a lot of associated changes in code. 

It also adds a pair of convenience constants, `FUND` and `FUND_FULL`, corresponding to fund abbreviation and full name, respectively.

This starts abstracting DCAF specific stuff out of the app, so we can ~ fulfill our destiny ~ and have a generalized solution.

This pull request makes the following changes:
* `patient.resolved_without_dcaf` to `patient.resolved_without_fund`
* `patient.dcaf_soft_pledge` to `patient.fund_pledge`
* add convenience constants

It relates to the following issue #s: 
* Bumps #999 

@DarthHater for review please!

### CODE REVIEW

File: app/models/pregnancy.rb [hunk 1]
```diff
@@ -1,55 +0,0 @@
-# Object representing pregnancy outcomes and status of a patient.
```
Comment (lomky) [inline app/models/pregnancy.rb]: Just cleanup here?

File: app/views/patients/_fulfillment.html.erb [hunk 2]
```diff
@@ -8,7 +8,7 @@
           <p>Please confirm that the clinic and pledge amounts are correct:</p>
           <ul>
             <li>Clinic: <%= @patient.clinic.try(:name) %></li>
-            <li>DCAF Pledge Amount: $<%= @patient.dcaf_soft_pledge %></li>
+            <li>DCAF Pledge Amount: $<%= @patient.fund_pledge %></li>
```
Comment (lomky) [inline app/views/patients/_fulfillment.html.erb]: `#{FUND}` here?

Comment (colinxfleming) [inline app/views/patients/_fulfillment.html.erb]: right you are!

File: app/models/pregnancy.rb (hunk 1 above)
Comment (colinxfleming) [inline app/models/pregnancy.rb]: Yeah, for some reason this didn't get deleted in the Great Mergening, so seemed appropriate to remove it in here.

**Pull request merged**